src/buildercore/terraform.py

```diff
@@ -486,24 +481,21 @@ def _generate_vcl_file(stackname, content, key, extension='vcl'):
         fp.write(str(content))
         return '${file("%s")}' % os.path.basename(fp.name)
 
-def render_gcs(context):
+def render_gcs(context, template):
     if not context['gcs']:
         return {}
 
-    return {
-        'resource': {
-            'google_storage_bucket': {
-                bucket_name: {
-                    'name': bucket_name,
-                    'location': 'us-east4',
-                    'storage_class': 'REGIONAL',
-                    'project': options['project'],
-                } for bucket_name, options in context['gcs'].items()
-            },
-        },
-    }
+    for bucket_name, options in context['gcs'].items():
+        template.populate_resource('google_storage_bucket', bucket_name, block={
```

Comment:
(very minor) maybe `block` is not the best name. But it was defined outside this PR.

Reply:
it's terminology coming from Terraform itself, although not "official" https://www.terraform.io/docs/configuration/terraform.html#description